sources/Context.cpp

```diff
@@ -105,6 +110,27 @@ Context::process() {
   return true;
 }
 
+void
+Context::filterDynModels() {
+  const auto& automatonsConfig = dynamicDataBaseManager_.assemblingDocument().dynamicAutomatons();
+  for (const auto& automaton : automatonsConfig) {
+    if (models_.models.count(automaton.id) == 0) {
+      continue;
+    }
+
+    const auto& modelDef = models_.models.at(automaton.id);
+    for (const auto& macroConnect : automaton.macroConnects) {
+      auto found = std::find_if(
```

Comment:
not efficient performance wise

Reply:
In all versions of the algorithm, we have to walk through that way. Besides, there is always a dozen connections maximum by model

Reply:
we will have to take a look at DFL performance at some point. OK for now